include/deal.II/multigrid/mg_constrained_dofs.h

```diff
@@ -311,60 +312,54 @@ MGConstrainedDoFs::initialize(
 
       // Loop through relevant cells and faces finding those which are periodic
       // neighbors.
-      typename DoFHandler<dim, spacedim>::cell_iterator cell = dof.begin(l),
-                                                        endc = dof.end(l);
-      for (; cell != endc; ++cell)
-        if (cell->level_subdomain_id() != numbers::artificial_subdomain_id)
-          {
-            for (auto f : cell->face_indices())
```

Comment:
From this line on, only the indentation changed because I removed the curly brace.